### ISSUE TITLE
iOS support

### ISSUE THREAD

**Issue by drets** [heutagogy/HeutagogyMobileApp#23]
Problem: currently we don't support iOS platform

Solution: It looks like for building mobile app for iOS we just need to go through https://github.com/alinz/react-native-share-extension readme notes for iOS platform and inject the main bundle of code to iOS index js file.

**Comment by hamsternik** [comment]
Yep, sure. In the near future, i don't have a free time to make iOS app, but I will do it soon.
